src/components/NoteDetails/NoteDetailsComponent.js

```diff
@@ -1,19 +1,44 @@
 import React, { Component } from 'react';
-import { StyledNoteNameEditor } from 'components/NoteDetails/NoteDetailsStyles';
+import {
+  StyledNoteNameEditor,
+  StyledDetailsScreen,
+  StyledNoteContentEditor,
+} from 'components/NoteDetails/NoteDetailsStyles';
 
 export class NoteDetailsComponent extends Component {
-  handleChange(e) {
-    this.props.onNoteNameChange(e.target.value);
+  handleChangeNoteName(e) {
```

Comment:
Please follow the naming used in the [React docs](https://reactjs.org/docs/forms.html#handling-multiple-inputs)

![image](https://user-images.githubusercontent.com/6686679/87776173-07056180-c830-11ea-8d9d-0d489b6d7dfb.png)

`handleChangeNoteName` -> `handleNoteNameChange`